documentation/userguide.rst

```diff
@@ -0,0 +1,99 @@
+.. figure:: figures/logo.jpg
+   :scale: 10 %
+   :alt: Logo
+
+User Guide
+==========
+
+The Water Network Tool for Resilience (WNTR) is an EPANET compatible Python package 
+designed to simulate and analyze resilience of water distribution networks.
+
+US EPA Disclaimer
+-----------------
+
+The U.S. Environmental Protection Agency through its Office of Research and Development funded and collaborated 
+in the research described here under an Interagency Agreement with the Department of Energy's Sandia National Laboratories.
+It has been subjected to the Agency's review and has been approved for publication. Note that approval does not signify that 
+the contents necessarily reflect the views of the Agency. Mention of trade names products, or services does not convey official 
+EPA approval, endorsement, or recommendation.  
+
+
+Sandia Funding Statement
+------------------------
+
+Sandia National Laboratories is a multimission laboratory managed and operated by National Technology and 
+Engineering Solutions of Sandia, LLC., a wholly owned subsidiary of Honeywell International, Inc., for the 
+U.S. Department of Energy's National Nuclear Security Administration under contract DE-NA-0003525.
+
+Contents
+--------
```

Comment:
remove "Contents" and "------".  The contents do not appear on that page

Reply:
Done